assets/src/design-system/components/menu/list/defaultListItem.js

```diff
@@ -37,15 +37,15 @@ const ActiveIcon = styled(CheckmarkSmall)`
   transform: translateY(-50%);
 `;
 
-const DefaultListItem = ({ option, isSelected, ...rest }, ref) => (
```

Comment:
updating the props internal here (`isSelected is now `selected`, `listLength` is now `listlength` so that we can quiet some of the eslint warnings about naming.

Reply:
Hm, is this new? The eslint naming warnings for these? Perhaps it's something that would need changing from the config?